File: mu_wang/9_10/Remove Element .py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        j= len(nums)-1
        if j<0:
            return 0
        for i in range(len(nums)-1,-1,-1):
            if nums[i] == val:
                nums[i], nums[j] = nums[j],nums[i]
                j-=1
        return j+1
                    

File: dayu/RemoveElement.java
public class RemoveElement {
    public int removeElement(int[] nums, int val) {
        int i, j = 0;
        for (i = 0; i < nums.length; i++){
            if (nums[i] != val) {
                nums[j++] = nums[i];
            }
        }
        return j;
    }
}
